test/e2e/k8s_util.go

```diff
@@ -0,0 +1,453 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package e2e
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	log "github.com/sirupsen/logrus"
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	v1net "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/remotecommand"
+
+	secv1alpha1 "github.com/vmware-tanzu/antrea/pkg/apis/security/v1alpha1"
+	secClient "github.com/vmware-tanzu/antrea/pkg/client/clientset/versioned/typed/security/v1alpha1"
+)
+
+type KubernetesUtils struct {
+	*TestData
+	podCache       map[string][]v1.Pod
+	ClientSet      *kubernetes.Clientset
+	SecurityClient *secClient.SecurityV1alpha1Client
+}
+
+func NewKubernetesUtils(data *TestData) (*KubernetesUtils, error) {
+	return &KubernetesUtils{
+		TestData:       data,
+		podCache:       map[string][]v1.Pod{},
+		ClientSet:      data.clientset.(*kubernetes.Clientset),
+		SecurityClient: data.securityClient.(*secClient.SecurityV1alpha1Client),
+	}, nil
+}
+
+// GetPod returns a pod with the matching namespace and name
```

Comment:
nits: s/pod/Pod/ and s/namespace/Namespace for consistency throughout the new code

but I know this is not introduced by you but must have been like that in the original netpol code

Reply:
Fixed those in comments. Not sure if the same needs to be done for ones in the log as well